docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -209,6 +209,73 @@ After you finish editing your machineSet, upload it to your OpenShift cluster:
 oc create -f sriov_machineset.yaml
 ```
 
+When creating SR-IOV worker ports, we recommend disabling the port security at a port level, as shown in the example above. This method is easier for the machine API to manage, and works nicely with the machine definitions generated by the installer automatically. However, if your architecture calls for you to create SR-IOV ports on a network with the port security disabled, you will have to make additional changes to your machine definiton. OpenStack forbids setting allowed address pairs or security groups on ports that have their port security disabled. The machines generated by the installer set security groups on the instance, which then applies those security groups to all ports attached as an interface to that instance. We also automatically apply allowed address pairs for the VIP ports to all ports created with the `networks` and `subnets` interfaces in the machine API. This means that in order to use a network with the port security disabled, you will have to define all of your ports using the `ports` interface in the machine API and specify the allowed address pairs and security groups on a port by port basis. To replace the port that attaches your node to the machines subnet, and by extension the kubernetes API, you will need to create a port that has the allowed address pairs for your API and ingress ports, and the worker security group. An example of how to do this can be seen in the first port defined in the verified example below. Then, the additional ports for your SR-IOV NICs can be defined as usual.
```

Comment:
s/definiton/definition/g

Reply:
rephrase and drop references to "we" and "you" 
for example: 
"we recommend disabling the port " => " disable port security" 
or
"However, if your architecture" => " If the architecture" 
In other words be more direct with statements, it will allow the paragraph to be smaller and clearer. 
also who is "we"? 

the `networks` and `subnets` interfaces in the machine API => the `networks` and `subnets` sections of the machineset.